browser/ui/brave_shields_data_controller.cc

```diff
@@ -19,6 +20,15 @@
 #include "content/public/browser/web_contents.h"
 #include "net/base/url_util.h"
 
+#if BUILDFLAG(ENABLE_IPFS)
+#include "brave/browser/ipfs/ipfs_service_factory.h"
+#include "brave/components/ipfs/ipfs_constants.h"
+#include "brave/components/ipfs/ipfs_utils.h"
+#include "chrome/common/channel_info.h"
```

Comment:
do not put header files inside guards that are not guarded by the flag

Reply:
removed ipfs related code as it will need to be addressed in a separate issue